apps/crn-server/test/data-providers/reminders.data-provider.test.ts

```diff
@@ -47,11 +49,31 @@ describe('Reminder Data Provider', () => {
     const fetchRemindersOptions: FetchRemindersOptions = { userId, timezone };
 
     describe('Research Output Published Reminder', () => {
-      test('Should fetch the reminder from squidex graphql', async () => {
+      test('Should fetch the reminders from squidex graphql', async () => {
         const result = await reminderDataProviderMockGraphql.fetch(
           fetchRemindersOptions,
         );
 
+        expect(result).toEqual({
+          total: 2,
+          items: [
+            getResearchOutputPublishedReminder(),
+            getResearchOutputDraftReminder(),
```

Comment:
this makes very little sense to me - why do we expect a draft RO for the published reminder test?

Reply:
this is probably an indication of an error of some sort - our mock server should only return a single RO for each of those two queries (that are part of a singe large query actually) and surely for the published reminders only one of them should be relevant?

Reply:
We have the following piece of code in squidex-graphql-client-with-server.mock
`    ApplicationQueries: () => ({
      queryResearchOutputsContents: [...new Array(1)],
    }),
    ` and both the draftROreminders and publishedROreminders use the queryResearchOutputsContents, so it normally returns for both of them.